policy/release/sbom_spdx/sbom_spdx_test.rego

```diff
@@ -337,6 +337,7 @@ test_attributes_multiple_external_refs if {
 
 _sbom_attestation := {"statement": {
 	"predicateType": "https://spdx.dev/Document",
+	"documentNamespace": "https://example.dev/spdxdocs/example-310683af-e9a0-4f66-a6a4-119352915b51",
```

Comment:
Shouldn't the `documentNamespace` field be within the `predicate` field?
Because the [tests are failing](https://github.com/enterprise-contract/ec-policies/actions/runs/15255739775/job/42957338652?pr=1397) with:
```
"msg": "SPDX SBOM at index 0 is not valid: (Root): documentNamespace is required"}
```